tools/syz-cover/syz-cover.go

```diff
@@ -54,6 +54,11 @@ func main() {
 	if err != nil {
 		tool.Fail(err)
 	}
+	subsystems := cfg.KernelSubsystem
```

Comment:
How would it affect the result's appearance?

Reply:
not related to this PR, so removed